afy/camera_selector.py

```diff
@@ -98,14 +100,17 @@ def select_camera(cam_frames, window="Camera selector"):
 
 
 if __name__ == '__main__':
-    with open('config.yaml', 'r') as f:
-        config = yaml.load(f, Loader=yaml.FullLoader)
+    parent_dir = os.path.basename(os.path.dirname(os.getcwd()))
+    file_location = ['Desktop', 'Downloads', 'Documents', parent_dir, '', 'Programs']
+    for loc in file_location:
+        yml_file_path = os.path.join(os.path.expanduser('~'), loc, 'config.yaml')
```

Comment:
You're still effectively only looking in the user's home folder, it needs to work regardless of where they have downloaded it. Joining parent_dir with the home folder also doesn't make much sense, but maybe this is a bug and you intended to do something else? I'm not quite sure if it's logical that someone would have "~/Downloads/config.yaml" while the actual application is somewhere else.